source/common/api/posix/os_sys_calls_impl.cc

```diff
@@ -187,5 +187,25 @@ SysCallSizeResult OsSysCallsImpl::write(os_fd_t sockfd, const void* buffer, size
   return {rc, rc != -1 ? 0 : errno};
 }
 
+SysCallSocketResult OsSysCallsImpl::accept(os_fd_t sockfd, sockaddr* addr, socklen_t* addrlen,
+                                           int flags) {
+  os_fd_t rc;
+
+#if defined(__linux__)
+  rc = ::accept4(sockfd, addr, addrlen, flags);
```

Comment:
What other flags can be passed in to this version of accept?

Reply:
SOCK_CLOEXEC, but as far as I understood, we're not currently using that.